docs/site/src/pages/style/color/color.md

```diff
@@ -0,0 +1,27 @@
+# Color
+
+[Color](https://material.io/guidelines/style/color.html) in material design is inspired by bold hues juxtaposed with muted environments, deep shadows, and bright highlights.
+
+## Color palette
+
+This color palette comprises primary and accent colors that can be used for illustration or to develop your brand colors.
+They’ve been designed to work harmoniously with each other.
+
+For instance, you can use the red color like so:
+```js
+import { red } from 'material-ui/styles/colors';
+
+const primary = red[500]; // #F44336
+```
+
+## Color schemes
+
+We've created javascript variables for every color used in the UI Color Palette. They are stored in `styles/colors.js.`
+
+These are links to some of the awesome tools that can be used to generate palettes.
```

Comment:
As far as I saw I just these links as good places to check out the preview of the color. I think this might be unnecessary as I can see the demo of the colors in the colors page itself. Or is there any other benefit in which it can be used with MUI that I am not able to see?